app/services/auth.server.ts

```diff
@@ -0,0 +1,49 @@
+import { User } from "@prisma/client";
+import { Authenticator } from "remix-auth";
+import { GitHubStrategy } from "remix-auth-github";
+import { GoogleStrategy } from "remix-auth-google";
+import { sessionStorage } from "~/services/session.server";
+import {
+  createOrLoginWithGithub,
+  createOrLoginWithGoogle,
+} from "~/shared/db/user.server";
+
+const url = `${
+  process.env.VERCEL_URL
+    ? `https://${process.env.VERCEL_URL}`
+    : "http://localhost:3000"
```

Comment:
3000 shouldn't be hardcoded, if we change it for the main server, it will need to automatically change here too

Reply:
Fixed!